link_shortener/templates/base.html

```diff
@@ -144,8 +144,17 @@
         color: #fff;
         background-color: #484d4d;
       }
-
     </style>
+
+    <!-- Global site tag (gtag.js) - Google Analytics -->
+    <script async src="https://www.googletagmanager.com/gtag/js?id={{ ga_id }}"></script>
```

Comment:
Aha, už vim proč to máš jako ID. Hele klidně configuj celý to URL, ta doména tady asi nikoho moc zajímat nebude

Reply:
Idecko pouzivam aj nizsie, takze dva env variables - pre URL a pre id, jo?

Reply:
Na to URL teda asi kašli. Buď celý jako DSN nebo jen ten ocas jako TRACKING_ID

Reply:
9ec275cf79802b1c4da9f79082e9df7f31cf8484